build.gradle

```diff
@@ -12,8 +12,8 @@ repositories {
 
 dependencies {
     testImplementation 'org.jetbrains.kotlin:kotlin-test:1.6.0'
-    implementation 'org.json:json:20171018'
     implementation 'com.google.code.gson:gson:2.8.9'
+    implementation "org.jetbrains.kotlin:kotlin-reflect:1.6.10"
```

Comment:
What does Kotlin Reflect do? It would be helpful if you could add changes to dependencies in the ReadMe with a short (1-2 sentence) description of what it does and/or why we need it.

Reply:
Kotlin Reflect allows me to examine the type of a property or a function at runtime. Gson only works with Java objects, and not with Kotlin; therefore, in order to obtain the Java class reference, I need to utilize `KClass`. For me to examine the class at runtime, I'll need to utilize Kotlin Reflection.

Reply:
I'll go ahead an make a `Dependency` section in the `README` and add those descriptions.